Mage.Sets/src/mage/cards/w/WishfulMerfolk.java

```diff
@@ -0,0 +1,65 @@
+package mage.cards.w;
+
+import mage.MageInt;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.common.continuous.BecomesCreatureSourceEffect;
+import mage.abilities.keyword.DefenderAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.game.permanent.token.TokenImpl;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author jmharmon
+ */
+
+public final class WishfulMerfolk extends CardImpl {
+
+    public WishfulMerfolk(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{U}");
+        this.subtype.add(SubType.MERFOLK);
+
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // Defender
+        this.addAbility(DefenderAbility.getInstance());
+
+        // {1}{U}: Wishful Merfolk loses defender and becomes a Human until end of turn.
+        //this.addAbility(new SimpleActivatedAbility(Zone.BATTLEFIELD, new WishfulMerfolkEffect(), new ManaCostsImpl("{1}{U}")));
+       this.addAbility(new SimpleActivatedAbility(Zone.BATTLEFIELD, new BecomesCreatureSourceEffect(new WishfulMerfolkToken(), "", Duration.EndOfTurn), new ManaCostsImpl("{1}{U}")));
```

Comment:
This should not turn Wishful Merfolk in a creature if it wasn't one, change the Power/Toughness, ...

Better split the ability in 2 parts (lose defender, and become human).